### PR TITLE
use publish script in jaxrs clients

### PR DESCRIPTION
**Pull request by ellisjoe**
@uschi2000

### CODE REVIEW

File: jaxrs-clients/build.gradle [hunk 1]
```diff
@@ -71,31 +72,3 @@ task checkLicenseInformation(dependsOn: 'shadowJar') << {
...
-
-// Publishing, taken mostly from gradle/publish.gradle.
-// This thing is super-brittle, be very careful when making changes.
-apply plugin: 'com.jfrog.bintray'
-apply plugin: 'maven-publish'
-apply plugin: 'nebula.maven-resolved-dependencies'
-
-publishing.publications {
-    shadow(MavenPublication) {
-        from components.shadow
```
Comment (ellisjoe) [inline jaxrs-clients/build.gradle:83]: unsure what this is doing

Comment (uschi2000) [reply jaxrs-clients/build.gradle:83]: jaxrs-clients is published as a shadowJar only

Comment (uschi2000) [reply jaxrs-clients/build.gradle:83]: hence the special casing here

**Comment by ellisjoe** [comment]
great. fixed and verified locally that the javadoc, sources, and shaded jar (with the normal name) are all published correctly

**Comment by uschi2000** [comment]
👎 POM advertises shaded dependencies. did you read me comment? ;)

```
// This thing is super-brittle, be very careful when making changes.
```

**Comment by uschi2000** [comment]
that looks ok now; cheers

**Comment by uschi2000** [comment]
can you rebase?

**Comment by ellisjoe** [comment]
rebased

**Pull request merged**